docs/install.md

```diff
@@ -1,77 +1,197 @@
+process these images as 'rawdisk' Evidence.
+
+# Instructions
+
+## GCP Project (Cloud Pub/Sub, Cloud Function, Cloud Datastore Setup)
+
+*   Create or select a Google Cloud Platform project on the [Google Developers
+    Console](https://console.developers.google.com)
+*   Enable [Cloud
+    Functions](https://console.cloud.google.com/apis/library/cloudfunctions.googleapis.com)
+*   Follow the
+    [instruction](https://cloud.google.com/pubsub/docs/quickstart-console) to:
+    *   Enable [Cloud
+        Pub/Sub](https://console.cloud.google.com/apis/library/pubsub.googleapis.com)
+    *   Create a new Pub/Sub topic and subscription **(pull type)**
+    *   Please take a note of the topic and subscription name for the
+        configuration steps
+*   Enable [Cloud
+    Datastore](https://console.cloud.google.com/apis/api/datastore.googleapis.com)
+    *   Go to Datastore in the cloud console
+    *   Hit the `Create Entity` button
+    *   Select the region that your data should be in. No need to create any
+        Entities after selecting your region
+*   Enable [Cloud Deployment Manager
+    V2](https://console.cloud.google.com/apis/library/deploymentmanager.googleapis.com)
+    for `gcloud`
+
+## Local Turbinia
+
+*   If Turbinia will run on local machines, jump to [Core Installation
+    Steps](#core-installation-steps)
+*   Otherwise, follow [GCP Turbinia](#gcp-turbinia)
+
+## GCP Turbinia
+
+The following is a one possible configuration and setup for Turbinia in GCP.
+This is still a rough process and future versions will be containerized.
+
+*   Create a new GCE server VM from a recent version of Debian or Ubuntu
+    *   This should work on other Linux flavors, but these are untested. Feel
+        free to test and fix them ;)
+*   Follow [Core Installation Steps](#core-installation-steps)
+*   Create a new image from the server VM's disk
+*   Create a new Instance Template using the newly created image
+*   Create a new Managed Instance Group from the newly created Instance Template
+*   Create a new GCS bucket and create new directories for `scripts` and
+    `output/logs`
+*   Mount your GCS bucket on your server VM
+*   cp `turbinia/tools/gcp_init/*.sh` into your locally mounted copy of
+    `$your_bucket/scripts`
+*   Edit the variables in `scripts/start-wrapper.sh` and
+    `scripts/start-turbinia-common.sh` as appropriate
+*   ***NOTE:*** Please note that the `start-wrapper.sh` script has a
+    `GOOGLE_APPLICATION_CREDENTIALS` environment var in the middle of the script
+    that needs to be updated
+*   In your worker VM, add a new custom metadata key `startup-script-url`
+    pointing to `gs://$your_bucket/scripts/start-wrapper.sh`
+*   Upon start, your VM should mount your GCS Bucket, and copy the start scripts
+    into the home directory of the Turbinia user and will then start the
+    Turbinia worker
+*   If you are running in GCP, you may also want to install [GCS
```

Comment:
This section already assumes that we're running in GCP, so I think we can remove the first part up to the comma.  Also, this is assumed to be installed at the steps above when referencing the scripts, so maybe move this to above the first mention of "GCS Bucket" in this section?

We could also add the note here with something like: "We're using GCS FUSE here for convenience to keep scripts and log files, but this isn't strictly necessary for Turbinia to run if you have an alternate means to save logging data".

Reply:
Done